cypress.d.ts

```diff
@@ -192,13 +192,13 @@ declare global {
 			/**
 			 * Delete all emails from the mailbox.
 			 */
-			mailpitDeleteAllEmails(): Chainable<void>;
+			mailpitDeleteAllEmails(): Chainable<Response<void>>;
```

Comment:
Changing the return signature is a breaking change which is something not needed at the moment.

Reply:
I changed the signature here because the implementation im MailpitCommands.ts in Version 1.3 returns	Cypress.Chainable<Cypress.Response<void>> so there was a mismatch. There was a mismatch between declaration and implementation. I didn't changed the implementation here. I can undo the im´´declaration change but the implementation schould then be changed to return the correct type?

`	mailpitDeleteEmailsBySearch(query: string): Cypress.Chainable<Cypress.Response<void>> {
		return cy.request({
			method: "DELETE",
			url: this.mailpitUrl("/v1/search"),
			qs: {
				query: query,
			},
			auth: this.auth,
		});
	}`

Reply:
Is it safe to have this missmatch?

Reply:
Let me check.